### ISSUE TITLE
User without access to primary site getting access errors

### ISSUE THREAD

**Issue by gglnx** [verbb/navigation#250]
**Description**

If an user isn't allowed to edit the primary site and trying to edit a navigation from the list of all navigations, they gettings an access error (»Navigation not enabled for site: PRIMARY SITE«). I think it's related to this commit: https://github.com/verbb/navigation/commit/7ed116e857febe74bd66060304ab619b936f7895

**Steps to reproduce**
1. Create user without access to primary site
2. Try to open a navigation from the list of all navigations

**Additional info**
- Plugin version: 1.4.14 
- Craft version: 3.6.8
- Multi-site: Yes
- Propagate nodes: No

**Comment by engram-design** [comment]
Yep, this was related to https://github.com/verbb/navigation/issues/245.

Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-3 as 1.4.14",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.15](https://github.com/verbb/navigation/releases/tag/1.4.15)
